aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -603,7 +603,7 @@ Resources:
           #!/bin/bash -x
           <%=indent(erb_file(BOOTSTRAP_CHEF,
             resource_id: daemon,
-            node_name: frontends ? '$ENVIRONMENT-daemon' : '$STACK',
```

Comment:
In production, aren't these values different?

```
ENVIRONMENT: production
STACK: autoscale-prod
```

Reply:
Yup. I’ll need to handle the special case for production before merging.